demos/pc/windows/common/application_code/main.c

```diff
@@ -217,6 +219,14 @@ void vApplicationIPNetworkEventHook( eIPCallbackEvent_t eNetworkEvent )
             /* Initialize AWS system libraries */
             SYSTEM_Init();
 
+            if ( AwsIotNetworkManager_Init( ) != pdPASS )
```

Comment:
assert true instead of logging and letting it go

Reply:
Fixed